docs/pages/repo/docs/handbook/prisma.mdx

```diff
@@ -0,0 +1,250 @@
+import { Tabs, Tab } from '../../../../components/Tabs'
+import Callout from '../../../../components/Callout'
+
+# Using Prisma with Turborepo
+
+[Prisma](https://www.prisma.io/) is an extremely popular ORM with automated migrations, type safety and integrated tooling. Using it with Turborepo can cut time you spend generating code, and easily make sure your generated Prisma code is always up-to-date.
+
+## Guide
+
+This guide shows you how to:
+
+1. Set up Prisma in a monorepo
+2. Handle migration and code generation scripts
+3. Cache those scripts with Turborepo
+4. Ensure that they're always run whenever `dev` or `build` is run
+
+If you've already got Prisma set up in your database, you can skip to [step 4](#4-setting-up-the-scripts).
+
+### 1. Create your Monorepo
+
+If you don't have an existing project, use our [quickstart](/repo/docs/getting-started/create-new) to create a new monorepo.
+
+### 2. Add a new `database` package
+
+Create a new folder called `database` inside packages with a `package.json` inside:
+
+```json filename="packages/database/package.json"
+{
+  "name": "database",
+  "dependencies": {
+    "@prisma/client": "latest"
+  },
+  "devDependencies": {
+    // Replace "latest" with the latest version
+    "prisma": "latest"
+  }
+}
+```
+
+If you're using `pnpm`, you should add a file at the root called `.npmrc`:
+
+```txt filename=".npmrc"
+public-hoist-pattern[]=*prisma*
+```
+
+Run your package manager's install step to install the new dependencies.
+
+### 3. Run `prisma init`
+
+`cd` into `packages/database`:
+
+```bash
+cd packages/database
+```
+
+Run `npx prisma init`.
```

Comment:
Will this command differ for the package manager installed or is it agnostic?

It would be convenient to run pnpm prisma init from the root, it requires the following script to the root’s package.json:

the single `prisma` script in <root>/package.json:
```json
{
  scripts: {
    "prisma": "pnpm --filter database exec prisma"
  }
}
```


I personally like this trade off of the added script scoped to the database package for saving the cd and thereby manking it easier to get started.

Reply:
I think this is cool, but probably too opinionated to include in our docs.